### DISCUSSION TITLE
Getting Not Found While Trying To Login With Github

### DISCUSSION THREAD

**Discussion by moseskamau338** [Thinkmill/keystatic#1223] [Q&A]
Hi.

Am having a weird issue where, I have configured Keystatic to work with Github and it works perfectly locally.
Now after deploying with Node, the site loads so does the `/keystatic` page.

When I click `Login with Github`, I get a screen that simply says `Not Found`
When I try the same login callback locally, I am authenticated properly.

What can I be missing? I really need help please.

**Comment by moseskamau338** [comment]
@simonswiss, I'd appreciate your insight and wisdom here. Thanks.

**Reply by birtles** [reply]
I'm hitting the same issue. I'm deploying with Astro and SST. So far as I can tell, it's hitting the Lambda for the server-side code but returning "Not found" from the Lambda.

I think what I really want to know is how to add logging to see if the problem is in the Astro integration or in Keystatic itself, somehow.
